### PR TITLE
Fix Size tracking URLs

### PR DESCRIPTION
**Pull request by gbirke**
The path separator was missing for tracking URLs.

### CODE REVIEW

File: shared/track_size_issues.js [hunk 1]
```diff
@@ -37,11 +37,11 @@ const trackSizeIssues = function( bannerElement, trackingLink, trackRatio ) {
         window.outerWidth + 'x' + window.outerHeight
     ].join( '--' );
 
-    var trackUrl = trackingLink + resolutions;
+    var trackUrl = trackingLink.replace( /\/$/, '' ) + '/' + resolutions;
```
Comment (JeroenDeDauw) [inline shared/track_size_issues.js:40]: What does this regex do? Remove single slashes?

File: shared/track_size_issues.js [hunk 2]
```diff
@@ -37,11 +37,11 @@ const trackSizeIssues = function( bannerElement, trackingLink, trackRatio ) {
...
     bannerElement.append( $( '<img id="WMDE_Banner-track-sizes" width="1" height="1">' ).attr( 'src', trackUrl ) );
-}
+};
```
Comment (JeroenDeDauw) [inline shared/track_size_issues.js:42]: Not part of this PR: like this there are two "tracking url" variables. Looks like the trackUrl one can be avoided by inlining.

File: shared/track_size_issues.js (hunk 1 above)
Comment (gbirke) [reply shared/track_size_issues.js:40]: Yes, it removes single slashes. Currently, `trackingLink` has none, but this might change in the future so I "normalize" the URL to be able to be able to always add another path element separated with a slash.
